src/view.tree.js

```diff
@@ -119,17 +134,43 @@ TreeView.prototype.show = function(repo, treeData) {
 
 TreeView.prototype.syncSelection = function() {
   var tree = this.$view.find('.octotree_view_body').jstree(true)
-    , path = location.pathname
+    , path = decodeURIComponent(location.pathname)
+    , recursiveLoad = this.store.get(STORE.RECURSIVE)
```

Comment:
duplicated `recursiveLoad = this.store.get(STORE.RECURSIVE)` with function `show` but both of them will be called from `octotree.js` and the store value `RECURSIVE` can be changed after saving options in setting so I decided to make this duplication.